docs/API.md

```diff
@@ -827,6 +850,24 @@ set, based on MSS value. For desired result, configure MSS first.***
 - **[GET]** - Size of the unacknowledged data in send buffer.
 ---
 
+| OptName               | Since | Binding | Type  | Units  | Default  | Range  |
+| --------------------- | ----- | ------- | ----- | ------ | -------- | ------ |
+| `SRTO_SNDDROPDELAY`   | 1.3.2 | pre     | `int` | ms     | 0        |        |
+
+- **[SET]** - sets the extra delay for the decision of the data sender to make
```

Comment:
- **[SET]** - Sets an extra delay before TLPKTDROP is triggered on the data sender. TLPKTDROP discards packets reported as lost if it is already too late to send them (the receiver would discard them even if received).
The total delay before TLPKTDROP is triggered consists of the LATENCY (defined by the receiver), plus `SRTO_SNDDROPDELAY`, plus 2 * the ACK interval (default = 20 ms).
`SRTO_SNDDROPDELAY` extends the tolerance for retransmitting packets at the expense of more likely retransmitting them uselessly. To be effective, it must have a value greater than 1000 - `SRTO_PEERLATENCY`.

Reply:
I just changed this "(defined by the receiver)" into (`SRTO_PEERLATENCY`). It's important because it's about the latency in THIS direction, and it's controlled by this very option on the sender machine, where actually the described `SRTO_SNDDROPDELAY` option matters.